lib/fake_braintree/client_token.rb

```diff
@@ -0,0 +1,14 @@
+module FakeBraintree
+  class ClientToken
+    def self.generate(options = {})
+      root_url = "http://localhost:#{ENV['GATEWAY_PORT']}"
+      unencoded_client_token = {
+        clientApiUrl: "#{root_url}/merchants/merchant_id/client_api",
+        authUrl: 'http://auth.venmo.dev:9292',
+        assetsUrl: root_url,
+        authorizationFingerprint: options['customer_id']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.